_inc/client/components/forms/index.jsx

```diff
@@ -16,6 +17,16 @@ export const FormFieldset = React.createClass( {
 	render: function() {
 		return (
 			<fieldset { ...omit( this.props, 'className' ) } className={ classnames( this.props.className, 'jp-form-fieldset' ) } >
+				{
+					this.props.support
```

Comment:
iirc we used to write ternaries like

```
  a ?
  b :
  c
```

Both `?` and `:` at the end of expressions